car/src/main/kotlin/Vehicle.kt

```diff
@@ -17,11 +17,4 @@ class Vehicle(val motor: Motor){
         return motor.useFossilFuel() == true || motor.useFlammelFuel() == true
```

Comment:
Dica: quando nós mexemos com variaveis ou funções que retornam boolean, nós nem precisamos precisamos utilizar o `==`, pois o valor `true` e o `false` já ativam o if da forma esperada
Então nessa linha você poderia tirar os `==` que ele continuaria com o mesmo comportamento, e fica mais facil de ler :smile: 
```suggestion
        return motor.useFossilFuel() || motor.useFlammelFuel()
```